### ISSUE TITLE
Statistics are missing in the UI

### ISSUE THREAD

**Issue by michal-plebanski-ideals** [reportportal/reportportal#2206]
**Describe the bug**

Statistics are missing in the launch tab of the UI, starting from a particular launch each consecutive job does not have statistics.

What we observe:

1. Big number of exceptions related to RabbitMQ, might be related to already existing issue https://github.com/reportportal/reportportal/issues/1454

```console
{"javatime": "2024-01-04 18:26:11,375", "level": "DEBUG", "source": "com.epam.ta.reportportal.ws.rabbit.AsyncReportingListener", "message": "exception : com.epam.ta.reportportal.exception.ReportPortalException, message : Test Item 'c9d5d0fd-f0ef-4dc6-a904-435c582c1943' not found. Did you use correct Test Item ID?,  cause : "}
{"javatime": "2024-01-04 18:26:11,375", "level": "WARN", "source": "o.s.amqp.rabbit.listener.ConditionalRejectingErrorHandler", "message": "Execution of Rabbit message listener failed.norg.springframework.amqp.rabbit.support.ListenerExecutionFailedException: Listener threw exceptionn	at org.springframework.amqp.rabbit.listener.AbstractMessageListenerContainer.wrapToListenerExecutionFailedExceptionIfNeeded(AbstractMessageListenerContainer.java:1778)n	at org.springframework.amqp.rabbit.listener.AbstractMessageListenerContainer.doInvokeListener(AbstractMessageListenerContainer.java:1724)n	at org.springframework.amqp.rabbit.listener.AbstractMessageListenerContainer.actualInvokeListener(AbstractMessageListenerContainer.java:1594)n	at org.springframework.amqp.rabbit.listener.AbstractMessageListenerContainer.invokeListener(AbstractMessageListenerContainer.java:1571)n	at org.springframework.amqp.rabbit.listener.AbstractMessageListenerContainer.doExecuteListener(AbstractMessageListenerContainer.java:1562)n	at org.springframework.amqp.rabbit.listener.AbstractMessageListenerContainer.executeListener(AbstractMessageListenerContainer.java:1506)nCaused by: org.springframework.amqp.AmqpRejectAndDontRequeueException: nulln	at com.epam.ta.reportportal.ws.rabbit.AsyncReportingListener.onMessage(AsyncReportingListener.java:186)n	at org.springframework.amqp.rabbit.listener.AbstractMessageListenerContainer.doInvokeListener(AbstractMessageListenerContainer.java:1720)nat org.springframework.amqp.rabbit.listener.AbstractMessageListenerContainer.actualInvokeListener(AbstractMessageListenerContainer.java:1594)n	at org.springframework.amqp.rabbit.listener.AbstractMessageListenerContainer.invokeListener(AbstractMessageListenerContainer.java:1571)n	at org.springframework.amqp.rabbit.listener.AbstractMessageListenerContainer.doExecuteListener(AbstractMessageListenerContainer.java:1562)n	at org.springframework.amqp.rabbit.listener.AbstractMessageListenerContainer.executeListener(AbstractMessageListenerContainer.java:1506)nCaused by: com.epam.ta.reportportal.exception.ReportPortalException: Test Item 'c9d5d0fd-f0ef-4dc6-a904-435c582c1943' not found. Did you use correct Test Item ID?n	at com.epam.ta.reportportal.core.item.impl.StartTestItemHandlerImpl.lambda$startChildItem$3(StartTestItemHandlerImpl.java:146)n	at java.base/java.util.Optional.orElseThrow(Optional.java:408)n	at com.epam.ta.reportportal.core.item.impl.StartTestItemHandlerImpl.startChildItem(StartTestItemHandlerImpl.java:146)n	at com.epam.ta.reportportal.core.item.impl.StartTestItemHandlerImpl$$FastClassBySpringCGLIB$$17d020e.invoke(<generated>)n	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)n	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:783)n"}
{"javatime": "2024-01-04 18:26:11,385", "level": "ERROR", "source": "com.epam.ta.reportportal.ws.rabbit.AsyncReportingListener", "message": "Dropping on maximum retry limit request of type = FINISH_TEST, for target id = 5fabfc99-c504-4742-83b7-f7d6a39282c0 "}
```

2. Big number of message in rabbit reporting.dlq of type "com.epam.ta.reportportal.core.configs.rabbit.DeserializablePair", example payload of rejected message:
```json
{"left":{"itemUuid":"0d790837-7e65-42bc-aa65-02bb9abfc9f8","uuid":"06ae3943-12aa-4004-86ed-edf569bf1e6a","launchUuid":"e67dbaf7-5ed7-45db-b157-6dc00ea88954","time":1703311253225,"message":"06:00:53.225 [TestNG-tests-1] INFO  - Initial browser configuration setup\n","level":"INFO"},"right":null}
```

3. Everything started with successful API call to 
```
52.58.238.50 - - [03/Jan/2024:19:16:43 +0000] "PUT /api/v2/<redacted>/item/a254c931-fc9a-465e-b1a7-e924664850f4 HTTP/1.1" 200 104 "-" "okhttp/3.14.9" 475 0.006 [reportportal-reportportal-api-headless] [] 10.203.46.114:8585 104 0.006 200 1241863e0bc0e3c86f43adfb11670d50
```

and immediately after that we observed first log with "Did you use correct Test Item ID" message, and after that we see multiple such logs every minute.

The item with such uuid exists in the database table "test_item".
 
**Expected behavior**

Statistics should show up for each launch in the UI

**Screenshots**

<img width="1103" alt="Screenshot 2024-01-04 at 19 48 45" src="https://github.com/reportportal/reportportal/assets/139214213/10cdcf6b-3077-4f48-9253-ddbb3fe4f9c6">
<img width="1472" alt="Screenshot 2024-01-04 at 18 28 53" src="https://github.com/reportportal/reportportal/assets/139214213/0c524fe5-9693-49e3-814d-6f8d28ab2331">
<img width="1472" alt="Screenshot 2024-01-04 at 18 30 32" src="https://github.com/reportportal/reportportal/assets/139214213/36e1251c-b3d5-42a2-971e-3e99a4d7dcb6">

**Versions:**

 - MacOS Sonoma 14.1.2, Google Chrome 120.0.6099.129
Current version:API Service: 5.10.1;Index Service: 5.10.0;Jobs Service: 5.10.0;Authorization Service: 5.10.0;Service UI: 5.10.0;

**Comment by MaryiaRymarchyk** [comment]
Hi @michal-plebanski-ideals  it looks like the issue not in the statistics itself, but you don't have reported data inside the launch. Can you see anything inside the launch without statistics - testcases/logs ?

**Comment by michal-plebanski-ideals** [comment]
@MaryiaRymarchyk I'm unable to provide details you are asking for because we rolled back the DB. 

What we believe is the breaking change is the enablement of logging for Selenide, after removal of below configuration we do not have any problems for 2 weeks in a row now:

```
        SelenideLogger.addListener("Report Portal logger", new ReportPortalSelenideEventListener());
```

our logger config: 
```
    <appender name="RP" class="com.epam.reportportal.logback.appender.ReportPortalAppender">
        <encoder>
            <pattern>%d{HH:mm:ss.SSS} [%t] %-5level - %msg%n</pattern>
            <pattern>[%t] - %msg%n</pattern>
        </encoder>
    </appender>
...
    <logger name="RP" level="TRACE" additivity="false">
        <appender-ref ref="RP"/>
    </logger>
```

and version: 
```
            "com.epam.reportportal:logger-java-selenide:5.1.4"
```

please could you let me know if our configuration is correct given the serialization errors posted above?

To summarize: we do not see problems with missing statistics in every launch when we disable selenide logger and clear the DB (we must clear DB because it's not possible to recover from this state), weird but thats how it is, we did not make any other changes in meantime. Any advice on how to enable Selenide logs is appreciated

```
{"left":{"itemUuid":"0d790837-7e65-42bc-aa65-02bb9abfc9f8","uuid":"06ae3943-12aa-4004-86ed-edf569bf1e6a","launchUuid":"e67dbaf7-5ed7-45db-b157-6dc00ea88954","time":1703311253225,"message":"06:00:53.225 [TestNG-tests-1] INFO  - Initial browser configuration setup\n","level":"INFO"},"right":null}
```
